### PR TITLE
Make column schema for filters

### PR DESCRIPTION
**Pull request by ricardolsmendes**
**- What I did**
Added features that allow the Sisense Connector to synchronize Dashboard and Widget filters as schema columns in Google Data Catalog.

**- How I did it**
Added the `__make_schema_for_dashboard()` and `__make_schema_for_widget()` methods to the `scrape.DataCatalogEntryFactory` class.

**- How to verify it**
Run the unit tests.

**- Description for the changelog**
Added features that allow the Sisense Connector to synchronize Dashboard and Widget filters as schema columns in Google Data Catalog.

PS: This PR is part of the effort to deliver #70.

### CODE REVIEW

File: google-datacatalog-sisense-connector/src/google/datacatalog_connectors/sisense/prepare/datacatalog_entry_factory.py [hunk 1]
```diff
@@ -81,8 +81,32 @@ def make_entry_for_dashboard(
...
+    @classmethod
+    def __make_schema_for_dashboard(
+            cls, dashboard_metadata: Dict[str, Any]) -> Optional[Schema]:
+
+        if not dashboard_metadata.get('filters'):
+            return
+
+        filters_column = datacatalog.ColumnSchema()
+        filters_column.column = 'filters'
+        filters_column.type = 'array of filters'
```
Comment (mesmacosta) [inline google-datacatalog-sisense-connector/src/google/datacatalog_connectors/sisense/prepare/datacatalog_entry_factory.py:97]: Type is an open field, but is there any benefit on specifying it as `array of filters`?

With the column name it's clear that you have filters there... unless I am missing something, I would suggest just sticking with standard types, maybe just `array`?

File: google-datacatalog-sisense-connector/src/google/datacatalog_connectors/sisense/prepare/datacatalog_entry_factory.py [hunk 2]
```diff
@@ -169,10 +193,50 @@ def make_entry_for_widget(
...
+        panels = widget_metadata['metadata']['panels']
+        filters = next((panel.get('items')
+                        for panel in panels
+                        if panel.get('name') == 'filters'), None)
+        if not filters:
+            return
+
+        filters_column = datacatalog.ColumnSchema()
+        filters_column.column = 'filters'
+        filters_column.type = 'array of filters'
```
Comment (mesmacosta) [inline google-datacatalog-sisense-connector/src/google/datacatalog_connectors/sisense/prepare/datacatalog_entry_factory.py:217]: same

**Review by mesmacosta — CHANGES_REQUESTED** [review]

File: google-datacatalog-sisense-connector/src/google/datacatalog_connectors/sisense/prepare/datacatalog_entry_factory.py (hunk 1 above)
Comment (ricardolsmendes) [reply google-datacatalog-sisense-connector/src/google/datacatalog_connectors/sisense/prepare/datacatalog_entry_factory.py]: I'm ok with `array`. Thanks!

File: google-datacatalog-sisense-connector/src/google/datacatalog_connectors/sisense/prepare/datacatalog_entry_factory.py (hunk 2 above)
Comment (ricardolsmendes) [reply google-datacatalog-sisense-connector/src/google/datacatalog_connectors/sisense/prepare/datacatalog_entry_factory.py]: Fixed

**Review by mesmacosta — APPROVED** [review]
LGTM!

**Pull request merged**